lib/ruby_lsp/requests/support/test_item.rb

```diff
@@ -0,0 +1,56 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Requests
+    module Support
+      # Represents a test item as defined by the VS Code interface to be used in the test explorer
+      # See https://code.visualstudio.com/api/references/vscode-api#TestItem
```

Comment:
Add a comment to mention that these test items are only for stuff discovered inside of files, but not files themselves or directories.